Makefile

```diff
@@ -2586,6 +2590,10 @@ $(REMOTE_CURL_PRIMARY): remote-curl.o http.o http-walker.o GIT-LDFLAGS $(GITLIBS
 	$(QUIET_LINK)$(CC) $(ALL_CFLAGS) -o $@ $(ALL_LDFLAGS) $(filter %.o,$^) \
 		$(CURL_LIBCURL) $(EXPAT_LIBEXPAT) $(LIBS)
 
+contrib/scalar/scalar$X: $(SCALAR_OBJECTS) GIT-LDFLAGS $(GITLIBS)
+	$(QUIET_LINK)$(CC) $(ALL_CFLAGS) -o $@ $(ALL_LDFLAGS) \
+		$(filter %.o,$^) $(LIBS)
+
```

Comment:
I am not so sure. The `INCLUDE_SCALAR` knob is not even part of this patch series, and even after introducing it, users can manually run `make contrib/scalar/scalar` and it will work _without_ specifying `INCLUDE_SCALAR=OhYes`.

Would you agree that it makes sense to restrict that knob to the meta rules like `all`, `docs`, etc?

Reply:
I was looking at this commit-by-commit and was expecting the `INCLUDE_SCALAR` to appear later. If you don't think that is valuable to include in this series, then ignore my comment.